src/openforms/js/components/admin/forms/ComponentSelection.stories.mdx

```diff
@@ -1,41 +0,0 @@
-import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
-
-import {FormContext} from 'components/admin/form_design/Context';
-
-import ComponentSelection from './ComponentSelection';
-
-<Meta title="Form design/ComponentSelection" component={ComponentSelection} />
-
-## Usage
-
-Given a context of available (Formio) fields/components, display them in a dropdown.
-
-Note that this component relies on the `FormContext` provider.
-
-export const Template = ({availableComponents, ...args}) => (
-  <FormContext.Provider value={{components: availableComponents}}>
-    <ComponentSelection {...args} />
-  </FormContext.Provider>
-);
-
-## Presentation
-
-<!-- FIXME: onChange doesn't seem to work in canvas -->
```

Comment:
What did this mean? The event seems to fire in the Default story.

Reply:
in the story it didn't show the "result" of the change event with the new value, but that's expected since `onChange` is a callback. I removed the dummy callbacks so that the change event now properly gets recorded with the actions.

In the MDX based stories (and SB 6) this was notably unreliable which is something SB 7 seems to have improved upon substantially.

Things will get better when we implement the new design and move to using Formik for our admin forms instead of rolling our own (form) state management.